testing/bucket_service.go

```diff
@@ -512,6 +512,10 @@ func FindBuckets(
 		name           string
 		organization   string
 		organizationID platform.ID
+
```

Comment:
Just a comment.
Could you please use a `findOptions platform.FindOptions` field here?